### PR TITLE
Update vacuumpump/ADOPTERS.yaml

### PR DESCRIPTION
**Pull request by OptidriveAdmin**

### CODE REVIEW

File: VacuumPump/ADOPTERS.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 description: This is a compilation list of the current adopters of the data model [Data model] of the Subject [Subject].  All fields are non mandatory. More info at https://smart-data-models.github.io/data-models/templates/dataModel/CURRENT_ADOPTERS.yaml
```
Comment (albertoabellagarcia) [inline VacuumPump/ADOPTERS.yaml:2]: Cannot be this a bit more precise?

**Review by OptidriveAdmin — COMMENTED** [review]
I prepared a new file. Do i have to create a new patch or how can i edit and improve the adopter.yaml file in this patch?

**Comment by albertoabellagarcia** [comment]
up to you both alternatives are nice

**Review by albertoabellagarcia — CHANGES_REQUESTED** [review]
Please include more info about the use case

**Review by albertoabellagarcia — APPROVED** [review]

**Pull request merged**